x/evidence/keeper/infraction_test.go

```diff
@@ -1,27 +1,21 @@
 package keeper_test
```

Comment:
Looking at it, I think the notion of "integration" tests should not be kept.
We should have mock tests in modules when it makes sense and test using an app when it makes sense as well.
Right now, when someone was looking at evidence, that person can have the false impression that infraction is not tested (I had that).

We may create guidelines on wait testing with a small app is necessary. Obviously https://github.com/cosmos/cosmos-sdk/pull/14695/files#r1081376406 should be resolved for this to work.